content/en/tracing/live_debugger/_index.md

```diff
@@ -0,0 +1,164 @@
+
+## Impact on performance and billing
+
+Enabling Live Debugger and Dynamic Instrumentation on a service does not trigger data capture or impact performance. Data capture only occurs when there are active Debug Sessions or dynamic instrumentations on that service.
+
+**Performance impact**: Datadog's agent-driven instrumentation ensures minimal impact on application performance; sampling logic, rate limits, and built-in budgets prevent runaway data capture. The system limits logs with variable capture to 1 per second, but permits logs _without_ variable capture a higher sampling rate (due to less overhead).
+
+**Pricing impact**: Logs captured by Datadog are all billed the same way, whether they are generated from Live Debugger or logger lines in your source code. With Live Debugger, the logpoints automatically expire after the set time period, limiting unnecessary data accumulation and costs. Monitor your [Datadog Plan & Usage page][21] for any unexpected increases after utilizing a new feature.
+
+## Using Live Debugger
+
+### Creating and using a Debug Session
+
+Debug Sessions let you inspect your code at runtime with auto-expiring logpoints. To create and use a Debug Session:
+
+1. Start a Debug Session from one of the following:
+    - On the [Live Debugger page][14], click **Create Debug Session**.
+    - (Requires Code Origins feature) In the [Trace Explorer][22], click on a trace to open the side panel, find the Code Origin section, and click **Start Debug Session**.
+2. Add the first logpoint to start the session.
+3. Add, remove, and modify logpoints within the session.
+
+Debug Sessions automatically expire after 48 hours. You can manually disable and re-enable both sessions and individual logpoints at any time.
+
+
+### Creating logpoints
+
+Logpoints are "non-breaking breakpoints" that specify where in the code to capture information, what data to include, and under what conditions. To add a logpoint for debugging:
+
+1. Go to the [Live Debugger page][14].
+2. Click **Create Debug Session**.
+3. Choose your service, environment, and select where in your code to place the first logpoint.
+4. Define a logpoint message template using the [Dynamic Instrumentation expression language][2].
+5. (Optional) Enable "Capture Variables" to collect all execution context (this feature is rate-limited to 1 execution per second).
+6. (Optional) Define a condition for when the logs should be emitted.
+
+**Note:** Some feature limitations may apply depending on the service's runtime language. Review the [runtime language-specific documentation][17] for more details.
+
+### Protecting sensitive data
+
+Live Debugger data might contain sensitive information, especially when using the "Capture Variables" option. To protect this data:
+
+1. Use Dynamic Instrumentation's built-in [sensitive data scrubbing][3] mechanisms.
+2. Use [Sensitive Data Scanner][15] to identify and redact sensitive information based on regular expressions.
```

Comment:
Both are mentioned on the first link, maybe just send them there? They'd need to follow the instructions on that page to use Sensitive Data Scanner successfully (see https://docs-staging.datadoghq.com/sarah.stonehill/live-debugger-docs/dynamic_instrumentation/sensitive-data-scrubbing/#customizing-sensitive-data-scanner)

Reply:
and we can modify that page to refer to both Dynamic Instrumentation and Live Debugging?